From 6e837676c81b164cf76d5a236e655efc4b185768 Mon Sep 17 00:00:00 2001
From: SrujanaChalla <schalla@marvell.com>
Date: Tue, 26 Mar 2019 12:13:34 +0530
Subject: [PATCH 088/386] crypto:cpt - Updated CPT_UCODE_VER_STR_SZ

Updated CPT_UCODE_VER_STR_SZ from 28 to 44 as per latest
Microcode release.

Change-Id: Ia6718ecf2ba266080d7308590d545f6472ce27b4
Signed-off-by: SrujanaChalla <schalla@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/6317
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Lukas Bartosik <Lukasz.Bartosik@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/crypto/cavium/cpt/common/cpt_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/cavium/cpt/common/cpt_common.h b/drivers/crypto/cavium/cpt/common/cpt_common.h
index 4b1ecad90420..b68da2a4cc7c 100644
--- a/drivers/crypto/cavium/cpt/common/cpt_common.h
+++ b/drivers/crypto/cavium/cpt/common/cpt_common.h
@@ -23,7 +23,7 @@
 #define CSR_DELAY		30
 
 /* Microcode version string length */
-#define CPT_UCODE_VER_STR_SZ	28
+#define CPT_UCODE_VER_STR_SZ	44
 #define TIME_IN_RESET_COUNT	5
 
 /* Completion code size and initial value */
-- 
2.17.1

